Colombia/transform_load_raw_dlt.py

```diff
@@ -537,6 +539,8 @@ def col_subnat_boost_silver_from_raw():
                 col("econ2").isin('ADQUISICIÓN DE BIENES', 'ADQUISICIÓN DE SERVICIOS', 'ADQUISICION DE BIENES Y SERVICIOS'), 'Goods and services'
             ).when(
                 col("econ2") == 'TRANSFERENCIAS DE CAPITAL', 'Capital expenditures'
+            ).when(
+                lower(col('econ2')).startswith("servicio") & ~col('econ1').contains('Adquisición de Activos Financieros'), 'Interest on debt'
```

Comment:
Ditto

Reply:
Interestingly, there was no filtering imposed on the _col_subnat_boost_silver_from_raw_ unlike the central dataset. 

1. I added the additional filtering for _col_subnat_boost_silver_from_raw_ upstream. 

2. The econ2 for subnat was somehow only in uppercase (unlike the central case), and I added the lower() function for both cases for better consistency.